### PR TITLE
Optimize code for run apps

### PR DESCRIPTION
**Pull request by kakaZhou719**

### CODE REVIEW

File: cmd/sealer/cmd/cluster/run.go [hunk 1]
```diff
@@ -240,6 +240,15 @@ func createNewCluster(imageEngine imageengine.Interface, cf clusterfile.Interfac
 
 	logrus.Infof("start to create new cluster with image: %s", clusterImageName)
 
+	cmds := infraDriver.GetClusterLaunchCmds()
+	appNames := infraDriver.GetClusterLaunchApps()
+
+	// merge to application between v2.ClusterSpec, v2.Application and image extension
+	v2App, err := application.NewV2Application(v2.ConstructApplication(cf.GetApplication(), cmds, appNames), imageSpec.ImageExtension)
```
Comment (Starnop) [inline cmd/sealer/cmd/cluster/run.go:247]: lazy init the v2app

Comment (kakaZhou719) [reply cmd/sealer/cmd/cluster/run.go:247]: i thought , if init app error ,return fast.

**Review by Starnop — APPROVED** [review]
LGTM

**Pull request merged**